docs/user_guide/readthedocs.md

```diff
@@ -2,6 +2,16 @@
 
 This theme comes with support for {{ rtd }}, a popular service for hosting documentation in the scientific Python community.
 
+## Version switcher
+
+Projects hosted on {{ rtd }} can use the {{ rtd }} supplied version switcher instead of the [version switcher that this theme provides](version-dropdown.rst).
+Its presence will be automatically detected by this theme, and placed in the `rtd-footer-container` node inside the primary sidebar.
+
+```{warning}
+The {{ rtd }} version switcher will be hidden any time the primary sidebar is hidden (see [this section](#layout-sidebar-primary) for discussion of when the primary sidebar might get hidden automatically and how to hide it purposely).
```

Comment:
hmm, the `[this section](#layout-sidebar-primary)` crossref isn't working.  It's meant to go to here: 

https://github.com/pydata/pydata-sphinx-theme/blob/a55cb93fef8cc57a85f9235dda784d0769aaa03d/docs/user_guide/layout.rst?plain=1#L299-L302

@VeckoTheGecko do you know how to do crossrefs correctly from a MystMD page to an rST page?  (I don't work with MystMD very often)